app/src/main/java/com/pajato/argus/Database.kt

```diff
@@ -0,0 +1,89 @@
+package com.pajato.argus
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.provider.BaseColumns
+
+/* Object that defines the table contents */
+object DatabaseEntry : BaseColumns {
```

Comment:
I'll be interested to know why this is not a class when we pair up.

Reply:
It would be a class in java; it would essentially be a static POJO, but because Kotlin has no "static" typing, it's an object, which serves a similar purpose. A more direct alternative would have been to make the DatabaseEntry class empty and have this object be it's "companion" object, but this is done with one less line of code, so I chose this way. This object serves as the current Database Schema. I could see an argument for making it the DatabaseHelper's companion object, but it makes more sense to me to have it defined and referenced as "DatabseEntry"